src/krbd.cc

```diff
@@ -764,3 +764,58 @@ int krbd_showmapped(struct krbd_ctx *ctx, Formatter *f)
 {
   return dump_images(ctx, f);
 }
+
+int krbd_is_image_mapped(struct krbd_ctx *ctx, const char *poolname, 
+                       const char *imgname, const char *snapname,
+                       std::ostringstream &mapped_info) {
+  struct udev_enumerate *enm;
+  struct udev_list_entry *l;
+  struct udev *udev = ctx->udev; 
+  const char *mapped_id, *mapped_pool, *mapped_image, *mapped_snap;
+  int r = 1;
```

Comment:
This is a little confusing to me. from the semantics, the "krbd_is_image_mapped()" should return a bool value, then we can use it as that:
```
if (krbd_is_image_mapped()) {
   // image is mapped.
} else {
  // image is not mapped.
}
```

That's okey if you want to return a int value instead of bool. But I found the ret-val of 1 means not mapped, but 0 means mapped. 

Then I think this function name should be renamed as "krbd_is_image_NOT_mapped()". 

what do you think? thanx